packages/completer/style/base.css

```diff
@@ -45,6 +45,7 @@
 }
 
 .jp-Completer-list {
+  overflow-x: hidden;
```

Comment:
This does not have any effect because there is a second `overflow-x` in line 53 (old line 52), which is what the failing lint job is warning about:

```
packages/completer/style/base.css
 53:3  ✖  Unexpected duplicate "overflow-x"  declaration-block-no-duplicate-properties
```

Can you instead change the line 52/53?

Reply:
Very good point. Apologies for this oversight.